DevoxxClientMobile/src/main/java/com/devoxx/DevoxxDrawerPresenter.java

```diff
@@ -93,35 +96,44 @@ public DevoxxDrawerPresenter() {
 
     @PostConstruct
     public void postConstruct() {
-        header.text.setText(DevoxxBundle.getString("OTN.DRAWER.CONF_NAME", getConferenceShortName(service.getConference())));
-        service.conferenceProperty().addListener((obs, ov, nv) ->
-            header.text.setText(DevoxxBundle.getString("OTN.DRAWER.CONF_NAME", getConferenceShortName(nv)))
-        );
+        header.text.setText(conferenceNameWithCountry(service.getConference()));
+        checkAndAddBadgesItem(service.getConference());
+        service.conferenceProperty().addListener((obs, ov, nv) -> {
+            header.text.pseudoClassStateChanged(PSEUDO_CLASS_VOXXED, nv.getEventType() == Conference.Type.VOXXED);
+            header.text.setText(conferenceNameWithCountry(nv));
```

Comment:
There is a slight problem with the listener: if you select the conference and immediately open the drawer you can still see the old name from the previous event. Not a big deal, but maybe the old name can be removed in advance.

Reply:
Joeri is working on some caching mechanism. Hopefully, the delay issue can be resolved with it.